src/main/java/stream/flarebot/flarebot/util/pagination/PagedEmbedBuilder.java

```diff
@@ -39,6 +39,17 @@ public PagedEmbedBuilder setCodeBlock(String codeBlock) {
         return this;
     }
 
+    /**
+     * Enables code blocks
+     * If you use {@link PagedEmbedBuilder#setCodeBlock(String)} it auto enables.
+     *
+     * @return this
+     */
+    public PagedEmbedBuilder setCodeBlock() {
+        this.codeBlock = "";
```

Comment:
null not empty string

Reply:
but null turns code blocks off. I chould add a separe varible in the builder it's self for if codeblock are on but do we really want that?